nav2_docking/opennav_docking/src/simple_charging_dock.cpp

```diff
@@ -133,11 +154,6 @@ void SimpleChargingDock::configure(
 
   if (use_external_detection_pose_) {
     dock_pose_.header.stamp = rclcpp::Time(0);
-    dock_pose_sub_ = node_->create_subscription<geometry_msgs::msg::PoseStamped>(
```

Comment:
If `!subscribe_toggle_`, I think we should create this, no?

Reply:
I think the code should move back up here. You also don't need to check `!detected_pose_sub_`

Reply:
@bkoensgen this was not completed :-)